### PR TITLE
Manually Bump SSP to v0.14.0

### PR DESCRIPTION
**Pull request by nunnatsa**
SSP now release its API as a separate package. This PR replaces the SSP
package by ssp/api, and bumps API to version v0.14.1

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Bump SSP to v0.14.1
```

### CODE REVIEW

**Review by tiraboschi — APPROVED** [review]

File: deploy/crds/scheduling-scale-performance00.crd.yaml [hunk 1]
```diff
@@ -14,7 +14,11 @@ spec:
     singular: ssp
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - additionalPrinterColumns:
+    - JSONPath: .status.phase
```
Comment (orenc1) [inline deploy/crds/scheduling-scale-performance00.crd.yaml:18]: hi @borod108 / @akrejcir ,
There is an error in the CRD definition for this property:
`JSONPath` --> `jsonPath`
```
error: error validating "_out/crds/scheduling-scale-performance00.crd.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec.versions[0].additionalPrinterColumns[0]): unknown field "JSONPath" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceColumnDefinition, ValidationError(CustomResourceDefinition.spec.versions[0].additionalPrinterColumns[0]): missing required field "jsonPath" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceColumnDefinition]
```

Comment (akrejcir) [reply deploy/crds/scheduling-scale-performance00.crd.yaml:18]: From where did you get this CRD?
The released CRD `olm-crds.yaml` has correct spelling `jsonPath`: https://github.com/kubevirt/ssp-operator/releases/tag/v0.14.0
And also I see correct spelling when searching through the SSP source code.

Comment (tiraboschi) [reply deploy/crds/scheduling-scale-performance00.crd.yaml:18]: @akrejcir here it is: https://github.com/kubevirt/ssp-operator/blob/6c9da3933bcc4176499955e79e3ab99bdccd381c/api/v1beta1/ssp_types.go#L98

Comment (akrejcir) [reply deploy/crds/scheduling-scale-performance00.crd.yaml:18]: I've fixed the csv-generator. Can you try the new release?

Comment (tiraboschi) [reply deploy/crds/scheduling-scale-performance00.crd.yaml]: done, thanks. Let's wait for CI results

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**